### ISSUE TITLE
any improvement to include the memory attention values?

### ISSUE THREAD

**Issue by chengmengli06** [waldenlqy/SAM#3]
https://github.com/waldenlqy/SAM/blob/main/SAM_codes/sam_model/sparse_attentive_memory_network.py#LL192C17-L192C34

Here the memory attention values are encoded by dnn, and then concat with the final output? any reason to include this? does it have big impacts on the final result?

**Comment by waldenlqy** [comment]
Nope. I remember it is the best with just the user interest vec, without the memory attention values concatenating with the final output.
